.changeset/famous-pants-agree.md

```diff
@@ -0,0 +1,6 @@
+---
+'@builder.io/sdk': patch
+'@builder.io/react': patch
+---
+
+Fix: Content API invocations will have `includeRefs` set to `false` by default and will not have `enrich` as part of its query-params
```

Comment:
`includeRefs` default should be set to `true` right?

Reply:
Yes, good catch, miss from my side